src/misc/ss.cpp

```diff
@@ -0,0 +1,101 @@
+//
+//  SS.cpp
+//  color_picker
+//
+//  Created by will on 2/3/19.
+//
+
+#include "ss.h"
+
+
+
+QString SS::QPushButtonInvisible() {
+    return {
+        "QPushButton{background : rgba(0,0,0,0); border : 0px; }"
+    };
+}
+
+QString SS::QSpinBox() {
+    return {
+        "QAbstractSpinBox { padding-right: 0px;    background: #292929; margin-right: -6px; }"
+        "QAbstractSpinBox, QLabel { color:rgba(255,255,255,.8); padding: 3px } QAbstractSpinBox::down-button, QAbstractSpinBox::up-button { background: rgba(0,0,0,0); border : 1px solid rgba(0,0,0,0); }"
```

Comment:
Try and properly format these to resemble something more readable, each targeted object can be on its own line